packages/mediapipe-task-text/lib/src/interface/tasks/language_detection/language_detector.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'language_detection.dart';
+
+/// {@template LanguageDetector}
+/// {@endtemplate}
```

Comment:
Can this empty template be removed? Or filled with intended doc? :)

Reply:
ha, whoops. Left myself a placeholder - time to fill it in!